lib/cliCommon.js

```diff
@@ -0,0 +1,30 @@
+const messages = {
```

Comment:
@jairo-bc why kebab case? According to conventions "...Your filename should be identical to your function’s name..."
https://github.com/airbnb/javascript#naming--camelCase-default-export

Though many files here ane named with kebab case it's against common JS conventions and I believe it should be gradually refactored in the future :)

Reply:
I thought camel case is only for components and react stuff. 
Looks like most cli's are referring to google's styleguide. https://google.github.io/styleguide/jsguide.html#file-name

Reply:
@jairo-bc hm... that would be strange to use google's styleguide in this project providing that [in the project's contributing.md it's specified that we use airbnb's styleguide here](https://github.com/bigcommerce/stencil-cli/blob/master/CONTRIBUTING.md#javascript-styleguide). I guess the initial developers of the project just used code styles from other languages (maybe Ruby, or PHP). Because I actually didn’t encounter JS projects where the kebab case is used.
And naming files similarly to the exported entity seems to be more handy and clear as for me :)